x/slashing/src/keeper.rs

```diff
@@ -87,6 +103,211 @@ impl<SK: StoreKey, PSK: ParamsSubspaceKey, SSK: SlashingStakingKeeper<SK, M>, M:
         self.slashing_params_keeper.set(ctx, genesis.params);
     }
 
+    pub fn handle_validator_signature<DB: Database>(
+        &self,
+        ctx: &mut BlockContext<'_, DB, SK>,
+        addr_bytes: prost::bytes::Bytes,
```

Comment:
I don't see all picture, but if you need then add `From<AccAdress> for ConsAdress` than using raw bytes

Reply:
I've used type from our domain type [`Validator` type](https://github.com/rumos-io/gears/blob/main/tendermint/src/types/proto/validator.rs#L39). I suggest that we can update both info and validator type. I mean non optional validator field with type containing an address (ConsAddr for now).

Reply:
I investigated this part and see that we can't change type because it is part of `tendermint` crate. The `AccAddress` and the other address types are placed in `gears`. It's not a good idea to create circular dependency. 

We can add some wrapper in `gears` . I think it is an extra work and I think it is better to save this type and this conversion. 

Alternative solution. We can add a type `Address` in tendermint that will wrap the bytes and validate them like `BaseAddress` in `gears`. After that we can add `impl From<Address> for ConsAddress in `gears`.

@joneskm, @NYBACHOK What do you think?

Reply:
Are there issue with moving `BaseAdress<>` into our tendermint crate?

Reply:
This has been resolved by introducing a new `address` crate.